geometry/point.h

```diff
@@ -181,6 +181,10 @@ struct TPoint {
 
   constexpr Scalar Dot(const TPoint& p) const { return (x * p.x) + (y * p.y); }
 
+  constexpr TPoint Left() const { return {-y, x}; };
```

Comment:
I really can't tell what the "left" or "right" of a point is. Does this really need to be a utility on the point?

Reply:
Just because I keep needing 90 degree rotations for things. Maybe it should be called RotateLeft/RotateRight? I'll remove it if you think it's too trivial.

Reply:
Yeah, I am leaning on the side of it being a bit too trivial. If only because it is not a standard utility and the "rotate by 90 degs" is hard to deduce from just the method name.

Reply:
sgtm, removed.